doc/source/magics.rst

```diff
@@ -43,12 +43,25 @@ For this example we'll use some simulated hardware.
 
     from ophyd.sim import det1, det2, motor1, motor2
 
+Before we begin, we must "label" this hardware. It is done for now
+(experimental) by setting the ``_ophyd_labels_`` attribute. For example:
+
+.. ipython:: python
+
+    det1._ophyd_labels_ = {'point_detectors', 'detectors'}
```

Comment:
The only way to pass labels is from class initialization. I wonder if we want to do

```py3
class MotorGroup(Device):
    x = Cpt(motor1 labels={'motors'})
    y = Cpt(motor2, labels={'motors'})
```